### PR TITLE
Copy ingest-pipeline to ingest_pipeline

### PR DESCRIPTION
**Pull request by ruflin**
For now, both exist. As soon as Kibana is updated, the old one can be removed.

### CODE REVIEW

**Comment by ruflin** [comment]
PR updated as https://github.com/elastic/package-registry/pull/581 was merged.

**Comment by mtojek** [comment]
I think we need to inform @jonathan-buttner about such changes.

**Comment by ruflin** [comment]
@mtojek Thanks for the reminder. Yes @jonathan-buttner you also need to update your repo.

File: packages/main.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import (
+	"path/filepath"
+	"strings"
+
+	"github.com/magefile/mage/sh"
+)
+
+func main() {
+	// Find all directories with ingest-pipeline
+	ingestPipelineDirs, _ := filepath.Glob("./*/dataset/*/elasticsearch/ingest-pi*")
```
Comment (mtojek) [inline packages/main.go:16]: @ruflin just noticed this file: I don't think it should be versioned. Is that on purpose?

Comment (ruflin) [reply packages/main.go:16]: @mtojek No, I forgot to remove it. I also need to take it out on the storage. On the storage layer I thought to keep it in for when I remove the old directories and then directly remove it. We could do the same here or remove it directly.

**Pull request merged**